lib/linguist/lazy_blob.rb

```diff
@@ -24,6 +24,10 @@ def initialize(repo, oid, name, mode = nil)
       @mode = mode
     end
 
+    def path
+      name
+    end
```

Comment:
I don't know where LazyBlob is used so I don't know what effect this would have.